### ISSUE TITLE
theft and extortion, Very Urgent

### ISSUE THREAD

**Issue by JBX3s** [ValveSoftware/Dota2-Gameplay#6083]
### Description

Hello 
Recently I have a huge problem with a user who constantly torments me and extorts items and money from me.

Tonight at 2:51 am, the person mentioned probably hacked into my account and without my consent, violating my privacy, bought an item from the steam store Slicer of the Depths
This item charged my account for PLN 169.30.
I am requesting a refund and removing the exchanged item from my account.
The situation is very urgent, please help in solving it as soon as possible or contact a steam support employee!!!

The matter is very important and urgent to be resolved, I expect a full refund for the purchased item and additionally help with the person harassing me and extorting money from me. If the situation does not clear up, I will have to inform the police and seek help there
![sc33](https://user-images.githubusercontent.com/121181308/208942542-4d70def1-a541-45ea-a76c-82216358975c.PNG)
![sc22](https://user-images.githubusercontent.com/121181308/208942558-935f0122-8b1f-43db-a418-99e425e66376.PNG)
![sc11](https://user-images.githubusercontent.com/121181308/208942568-763913bb-27a9-4ec9-94bf-d4e3d7d539c6.PNG)

### Screenshots

![sc22](https://user-images.githubusercontent.com/121181308/208942417-1b6a0991-03fc-4335-8ec3-1c14318513aa.PNG)
![sc33](https://user-images.githubusercontent.com/121181308/208942437-43b34eaa-040c-4e92-8ff4-31ccd2c7e64c.PNG)
![sc11](https://user-images.githubusercontent.com/121181308/208942455-5777021c-34be-4e4b-9308-5f0bca7cc145.PNG)

**Comment by shaileshgarg** [comment]
This is not right place for the issue, contact steam support and possibly change your password.
